pywbem/_subscription_manager.py

```diff
@@ -1190,14 +1200,27 @@ def _create_destination(self, server_id, dest_url, owned):
                         dest_inst = server.conn.GetInstance(dest_path)
                         self._owned_destinations[server_id][i] = dest_inst
                     return dest_inst
-            dest_path = server.conn.CreateInstance(dest_inst)
+            dest_path = server.conn.CreateInstance(dest_inst,
+                                                   namespace=server.interop_ns)
             dest_inst = server.conn.GetInstance(dest_path)
             self._owned_destinations[server_id].append(dest_inst)
         else:
             # Responsibility to ensure it does not exist yet is with the user
             dest_path = server.conn.CreateInstance(dest_inst)
```

Comment:
Here we should also add `namespace=server.interop_ns`.

Reply:
Done